app/schemas/medication.py

```
@@ -43,6 +46,20 @@ def _normalize_reminder_times(value):
 
 
+def _normalize_reminder_days(value):
+    """Shared validator body for reminder_days — deduplicates, sorts, validates 0-6."""
+    if value is None:
+        return None
+    if not isinstance(value, list):
+        raise ValueError("reminder_days must be a list of integers (0=Mon … 6=Sun)")
+    invalid = [d for d in value if not isinstance(d, int) or d not in VALID_REMINDER_DAYS]
+    if invalid:
+        raise ValueError(
+            f"reminder_days values must be integers 0–6 (Mon=0, Sun=6); invalid: {invalid}"
```

Ruff (0.15.18)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)